### ISSUE TITLE
pchtxt2cheat() cannot handle certain comments

### ISSUE THREAD

**Issue by Craftyawesome** [ppkantorski/libultrahand#4]
Attempting this patch using mod alchemist results in only one line. The ips conversion seems to work correctly due to being different code. 
[anisotropic filtering.pchtxt.txt](https://github.com/user-attachments/files/18191026/anisotropic.filtering.pchtxt.txt)

**Comment by ppkantorski** [comment]
I accepted the PR, did this fix the issue you described?  Thanks again btw.

**Comment by Craftyawesome** [comment]
No, that is completely separate. I just happened to find both issues at the same time and fixed what affected me more (and seemed easier)

**Comment by ppkantorski** [comment]
i'll have to test this.  sorry for the late response.  i'll see if i can ensure it strips the data properly

**Comment by ppkantorski** [comment]
think that I was able to fix it now.  but i'll have to update Ultrahand.  i'll lyk when its updated.

**Comment by ppkantorski** [comment]
should be resolved.  do lmk though if you still have issues

**Comment by Craftyawesome** [comment]
Is there any particular reason the fix commit also reverted my PR?

**Comment by ppkantorski** [comment]
did it get reverted? thats weird. could you submit a new PR again if possible? i have to see what you did again.

**Comment by ppkantorski** [comment]
ok i fixed it.  should be back
